aiida_common_workflows/workflows/relax/gpaw/workchain.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""Implementation of `aiida_common_workflows.common.relax.workchain.CommonRelaxWorkChain` for GPAW."""
+from aiida import orm
+from aiida.engine import calcfunction
+from aiida.plugins import WorkflowFactory
+
+from ..workchain import CommonRelaxWorkChain
+from .generator import GPAWCommonRelaxInputGenerator
+
+__all__ = ('GPAWCommonRelaxWorkChain',)
+
+
+@calcfunction
+def extract_from_array(array):
```

Comment:
```suggestion
def extract_forces_from_array(array):
```
This will make it easier to recognize in provenance graphs and `verdi process list` output